internal/cli/cli.go

```diff
@@ -30,8 +30,11 @@ type tiltCmd interface {
 
 func addCommand(parent *cobra.Command, child tiltCmd) {
 	cobraChild := child.register()
-	cobraChild.RunE = func(_ *cobra.Command, args []string) error {
-		return child.run(args)
+	cobraChild.Run = func(_ *cobra.Command, args []string) {
+		err := child.run(args)
+		if err != nil {
+			log.Fatalf("Error: %v", err)
```

Comment:
`log` isn't meant for user-facing output, use fmt.Fprintf/fmt.Printf and os.Exit

`log` is for when you have machine-readable logs that you might want json-formatted or timestamp-appended or whatever